DEVELOPMENT.md

```diff
@@ -52,46 +48,52 @@ Tests environments can be spun up using the following command.
 yarn test --watch
 ```
 
-### Browser environment (extension)
+### Integration environments
+
+Running an devtools in an integrated environment can be useful for working on/testing messaging features.
 
-Running an E2E version of the extension can be useful for times where a more integrated environment is required.
+#### Browser environment (extension)
 
-#### Start a build
+##### Start a build
 
 Run the following command to start a watched build.
 
 ```sh
 yarn dev:extension
 ```
 
-#### Load the extension
+##### Load the extension
 
 Load the built extension in [chrome](https://developer.chrome.com/extensions/getstarted#manifest) or [firefox](https://developer.mozilla.org/en-US/docs/Tools/about:debugging#Extensions).
 
-#### Visit a project
+##### Visit a project
 
-Start a project which uses the devtools exchange and navigate to it in your browser - or alternatively use the [live project](#Live-project).
+Visit the [live urql project](https://urql-devtools-exchange.netlify.app/) - or create an app which uses the devtools exchange and navigate to it in your browser.
 
-### Standalone environment (electron)
+#### Standalone environment (electron)
 
-Running an E2E version of the extension can be useful for times where a more integrated environment is required.
 
-#### Start a build
+##### Start a build
 
 Run the following command to start a watched build.
 
 ```sh
 yarn dev:electron
 ```
 
-#### Start the app
+##### Start devtools
 
 Run the following command in a separate shell to start the built app.
 
 ```sh
 yarn start
 ```
 
+##### Start an app
+
+There's a react native app that uses `@urql/devtools` [such as this one](https://github.com/kadikraman/UrqlTest).
```

Comment:
If you don't already have a React Native app that uses `@urql/devtools`, you could use [this example app](https://github.com/kadikraman/UrqlTest)

Reply:
```suggestion
If you don't already have a React Native app that uses @urql/devtools, you could use [this example app](https://github.com/kadikraman/UrqlTest)
```